patches/@metamask+open-rpc-docs-react+0.1.2.patch

```diff
@@ -21,12 +21,15 @@ index 354ce73..129371f 100644
 +            result: examplePairing.result.value
 +        }
 +        // const jsCode = `await window.ethereum.request(${JSON.stringify(methodCall, null, "  ")});`;
++        const serverUrl = methodName.indexOf('_') > -1
```

Comment:
Not sure if method name containing `_` is a good enough way to distinguish it... We may add something on either RPC side that could break this logic. Is there anything else we can use? Tags maybe?

Reply:
You're absolutely right, this is a **really fragile** way to make this check. We're well on our way to publishing [our own fork](https://github.com/stellar/open-rpc-docs-react) of the package, and this patch file will (thankfully) be out of the picture soon. So, it seemed like this quick-n-dirty bandaid might be sufficient just for the time-being.

We could definitely do a more robust check now, if you'd prefer, though. iirc the stellar-rpc doesn't use tags for _any_ of its methods, so that probably is a better way to go about it.

Reply:
Sounds good, if it's temporary let's just leave the glue there for now